src/api/store/media_library.py

```diff
@@ -196,7 +204,9 @@ def get_most_recent(self, args, context: Context):
             else:
                 query |= qObj
 
+        count = 0
         if not upper_limit and not lower_limit:
+            count = Media.objects.filter(query).distinct().count()
```

Comment:
change to count = images.count() two lines later?  Seems this might avoid some DB access